examples/docker_compose/docker-compose.yaml

```diff
@@ -0,0 +1,77 @@
+version: '3'
+services:
+  ############################
+  # Dapr Sidecar
+  ############################
+  echoapp-dapr:
+    image: "daprio/daprd:edge"
+    command: ["./daprd",
+      "-app-id", "echo-service",
+      "-app-port", "9004",
+      "-dapr-grpc-port", "50002",
+      "-dapr-http-port","3502",
+      "-log-level", "debug",
+      "-placement-host-address", "placement:50005",
+      "-components-path","/opt/config"]
+    depends_on:
+      - echoapp
+    network_mode: "service:echoapp"
+    volumes:
+      - ../config:/opt/config
+
+  ############################
+  # echo app
+  ############################
+  echoapp:
+    build: "../echo"
+    platform: wasi/wasm
+    runtime: "io.containerd.wasmedge.v1"
+    depends_on:
+      - placement
+      - redis
+    ports:
+      - 50002:50002
```

Comment:
It seems that the echo app only opens port 9004. Do we really need these port mappings for the container? @SAtacker

Reply:
Thanks, removed those!

Reply:
Hmm, can you test and make sure that we can remove all the port mappings? We do not even need 9004? 

We may not need it since we are not using host networking, but I'd like to make sure. Thanks.

Reply:
@SAtacker Just to make sure that you see the above comment. Thanks!